pycln.toml

```diff
@@ -20,5 +20,5 @@ skip_imports = [
   "win32pipe",
   "win32ras",
   "win32ui",
-  "winxpgui",
+  "winxpgui"
```

Comment:
That one I'd rather not, but it isn't currently configurable: https://github.com/macisamuele/language-formatters-pre-commit-hooks/issues/152